coursemapper-kg/concept-map/src/graph_db.py

```
@@ -54,6 +54,14 @@ def get_concept_by_name(self, material_id: str, concept_name: str):
         if len(records) == 0:
             return None
         return Node.from_dict(records[0]['n'])
+    
+    def update_concept_node(self, node):
```

Ruff (0.12.2)

Missing return type annotation for public function `update_concept_node`

Add return type annotation: `None`

(ANN201)